<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGameDialog</class>
 <widget class="QDialog" name="NewGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: darkCyan;</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="mediumModeRadioButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>Keskitaso</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">difficultyButtonGroup</string>
   </attribute>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="hardModeRadioButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>Vaikea</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">difficultyButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="twoPlayersRadioButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">playerAmountButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="easyModeRadioButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>Helppo</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">difficultyButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu LGC Sans Mono</family>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Muistipeli</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeMono</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>Vaikeustaso:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeMono</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>Pelaajien lukumäärä:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="threePlayersRadioButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">playerAmountButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="nextPushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>450</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Carlito</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(46, 52, 54);</string>
   </property>
   <property name="text">
    <string>SEURAAVA &gt;</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="fourPlayersRadioButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">playerAmountButtonGroup</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="difficultyButtonGroup"/>
  <buttongroup name="playerAmountButtonGroup"/>
 </buttongroups>
</ui>
